app/assets/stylesheets/sass/views/employees/_check.css.scss

```diff
@@ -36,14 +34,15 @@ body {
           width: 25rem;
           
           div.screen {
-            background-color: $normal-cyan;
-            border-radius: 1.3rem;
-            color: white;
-            font-size: 4rem;
-            height: auto;
-            padding-right: 0.4rem;
-            text-align: right;
-            width: 100%;
+            input[type=password] {
+              background: $normal-cyan;
+              color: white;
+              font-size: 4rem;
+              padding-right: 0.4rem;
+              text-align: right;
+              width: 100%;
+              border-radius: 1.3rem;
+            }
           }
 
```

Comment:
Check the indentation of this file, the recommended spacing is two.

Reply:
Indeed the actual indentation is two spacing